addon/components/virtual-each.js

```diff
@@ -170,11 +172,11 @@ const VirtualEachComponent = Component.extend(EventListenerMixin, DefaultAttrsMi
     this._super(...arguments);
 
     RSVP.cast(this.getAttr('items')).then((attrItems) => {
-      let items = Ember.A(attrItems);
+      const items = Ember.A(attrItems);
```

Comment:
The wrapper Ember.A here will also likely need to go, but will tackle that in a near future PR